modules/apps_api/spec/lib/notification_service_spec.rb

```diff
@@ -148,7 +148,7 @@
     # lib/va_notify/configuration.rb#initialize contains:
     # @notify_client ||= Notifications::Client.new(api_key, client_url)
     allow(Notifications::Client).to receive(:new).and_return(notification_client)
-    allow(notification_client).to receive(:send_email)
+    allow(notification_client).to receive(:send_email).and_return(true)
```

Comment:
any reason not to use `expect` here instead of `allow`? just to make sure the calls to those methods actually happen